### ISSUE TITLE
The logo chibi needs alt text

### ISSUE THREAD

**Issue by macdonst** [enhance-dev/enhance.dev#52]
I suppose the alt text should be "enhance logo" but I'm pulling for "chibi cherub" who I affectionately call "chub" in my head.
